pages/registries.py

```diff
@@ -37,16 +36,11 @@ def get_first_non_withdrawn_registration(self):
             except NoSuchElementException:
                 return result.find_element_by_css_selector('[data-test-result-title-id]')
 
-
-class EmberRegistrationDetailPage(GuidBasePage):
-    url_base = urljoin(settings.OSF_HOME, '{guid}')
-    identity = Locator(By.CSS_SELECTOR, '[data-test-registration-title]')
-    title = Locator(By.CSS_SELECTOR, '[data-test-registration-title]')
-
-
 class RegistrationDetailPage(GuidBasePage):
-    waffle_override = {'ember_registries_detail_page': EmberRegistrationDetailPage}
```

Comment:
We removed waffle_override because the Registries detail overview page has been 'emberized', so it's no longer needed.  ~Josh